server/item/creative/creative.go

```diff
@@ -48,27 +53,15 @@ func RegisterGroup(group Group) {
 	creativeGroups = append(creativeGroups, group)
 }
 
-// Items returns a list with all items that have been registered as a creative item. These items will
-// be accessible by players in-game who have creative mode enabled.
-func Items() []Item {
-	return creativeItemStacks
-}
-
-// RegisterItem registers an item as a creative item, exposing it in the creative inventory.
-func RegisterItem(item Item) {
-	creativeItemStacks = append(creativeItemStacks, item)
-}
```

Comment:
Not sure why this was removed, as I said on discord:
I dont really know how we should do this, we need a way for people to register their own groups but also inject into vanilla groups